RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+
+## On a fork
+
+Please follow these instructions carefully.
+Ensure that you name the branch precisely as `release-vX.Y.Z`
+since this nomenclature is crucial for obtaining the `future_version` in the changelog.
+Your attention to this specific branch naming convention is essential for accurate version tracking in the changelog.
+
+```shell
+export RELEASE_VERSION="X.Y.Z"
+git switch main
+git pull -r
+git switch -c release-v$RELEASE_VERSION
+
+bundle config set --local path vendor/bundle
+bundle config set --local with 'release'
+bundle install
+
+CHANGELOG_GITHUB_TOKEN="token_MC_tokenface" bundle exec rake changelog
+git commit -am "Release v${RELEASE_VERSION}"
+git push origin release-v$RELEASE_VERSION
+```
+
+Then open a PR, discuss and merge.
+
+## After the merge, as a maintainer on upstream
+
+```shell
+git switch main
+git pull -r
+git tag v$RELEASE_VERSION -m "v$RELEASE_VERSION"
+git push --tags
```

Comment:
We have this too in other places, but I feel it's broken for a bit of a nit pick reason.

If you have a merge commit then the history is very confusing because you end up tagging the merge commit, not the commit named "Release $version". Because of that I'm more a fan of a fast forward merge and tag that.

For Foreman releases I use https://github.com/theforeman/foreman-installer-modulesync/blob/master/bin/release-merge which calls https://github.com/theforeman/foreman-installer-modulesync/blob/master/bin/release-module to actually release. Note we don't have release actions to run after a push, so for Vox Pupuli it could be simplified.

This works well together with https://github.com/theforeman/foreman-installer-modulesync/blob/master/bin/release-pr to create the actual PR.

Reply:
hmm interesting approach. I don't really understand why i should do it this way, but I'm not against it. 

Why is it better to tag the "release-commit" instead the merge-commit? What improvement does it give me?

what is `git phr...`? and whats `addfork`?

Reply:
i will handle this outside of this PR.
i created an issue for this: https://github.com/voxpupuli/container_modulesync_configs/issues/10